syntax.d/v.ini

```diff
@@ -0,0 +1,14 @@
+name=V
+extensions=v
+highlight_numbers=true
+singleline_string_quotes="
+singleline_comment_start=//
+multiline_comment_delims=/*, */
+multiline_string_delim="
+; https://github.com/vlang/v/blob/master/doc/docs.md#types
+; https://github.com/vlang/v/blob/master/doc/docs.md#appendix-i-keywords
+keywords_1=as, asm, assert, atomic, break, const, continue, defer, else, embed, enum, false, fn, for, go, goto, 
+if, import, in, interface, is, lock, match, module, mut, none, or, pub, return, rlock, select, shared, sizeof, 
```

Comment:
The syntax unit test fails here: https://travis-ci.com/github/ilai-deutel/kibi/jobs/464005170
The config parser is very rudimentary, all the keywords must be on the same line